core/google/cloud/_helpers.py

```diff
@@ -646,7 +475,9 @@ def make_secure_channel(credentials, user_agent, host):
     # ssl_channel_credentials() loads root certificates from
     # `grpc/_adapter/credentials/roots.pem`.
     transport_creds = grpc.ssl_channel_credentials()
-    custom_metadata_plugin = MetadataPlugin(credentials)
+    http = httplib2.Http()
+    custom_metadata_plugin = AuthMetadataPlugin(
+        credentials, google_auth_httplib2.Request(http=http))
```

Comment:
No need for a code change, but

1. How do you see this changing with "any old transport"?
1. How soon until gRPC can use local-only tokens (i.e. no transport needed, just local crypto to sign JWTs)